lib/ui.rb

```diff
@@ -90,6 +98,13 @@ def warn(s)
       def error(s)
         STDERR.puts s
       end
+
+      private
+
+      def reset_line
+        print "\033[2K"    # Clear line content
```

Comment:
Unnecessary spacing detected.<br>Do not write to stdout. Use Rails' logger if you want to log.